nbs/01_datasets.ipynb

```diff
@@ -252,7 +252,8 @@
     "            if self.shuffle:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #57.                return self.collate_fn(batch_data)
</pre><p>Change to <code>return batch_data</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/cfnet/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>